build.sh

```diff
@@ -42,6 +42,23 @@
 
 #############################  End of CI variables  ##############################
 
+# Get some options
+while test $# -gt 0; do
+    case "$1" in
+        -f|--env-file)
+            shift
+            ENV_FILE=$1
+            shift
+            ;;
+    esac
+done
+
+ENV_FILE=${ENV_FILE:-.env}
+if [ -f "./$ENV_FILE" ]; then
+    echo "Reading env file $ENV_FILE"
+    . "./$ENV_FILE"
```

Comment:
this fail if `ENV_FILE` is in a sub-directory or specified as a full path

Reply:
got to resolve to full path.